aws/resource_aws_eks_cluster.go

```diff
@@ -38,6 +38,9 @@ func resourceAwsEksCluster() *schema.Resource {
 			Delete: schema.DefaultTimeout(15 * time.Minute),
 		},
 
+		SchemaVersion: 1,
+		MigrateState:  resourceAwsEksClusterMigrateState,
```

Comment:
Hey @ewbankkit 👋 Is there a particular reason why a state migration was implemented here? `TypeList` to `TypeSet` attribute conversions should not require these as the old indexes should be overwritten during refresh. Less is sometimes more! 😉 Thanks.